orchestrator/cli/generator/generator/migration.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019-2020 SURF.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+import subprocess  # noqa: S404
+from typing import Generator, Optional
+
+import structlog
+
+from orchestrator.cli.generator.generator.helpers import get_variable
+
+logger = structlog.getLogger(__name__)
+
+
+def create_migration_file(config: dict) -> Optional[str]:
+    create_migration_command = f'PYTHONPATH=. DATABASE_URI=postgresql://nwa:nwa@localhost/nwa-workflows python main.py db revision --message "Add {config["name"]}" --head=data@head'
+    result = subprocess.check_output(create_migration_command, shell=True, text=True)  # noqa: S602
```

Comment:
Is it possible to import the typer/click command and invoke it programmatically?